### ISSUE TITLE
Commit signature validation failure

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/dbft#15]
Shouldn't ever happen in local private net:
```
2020-01-30T15:46:42.709Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
2020-01-30T15:46:42.710Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
2020-01-30T15:46:42.711Z        DEBUG   dbft@v0.0.0-20200130105505-02c208d154bf/check.go:57     not enough to commit    {"count": 1}
2020-01-30T15:46:42.711Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
2020-01-30T15:46:42.711Z        DEBUG   dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:503     recovering finished cv=0/4 preq=0/0 presp=0/4 co=4/4
2020-01-30T15:46:42.711Z        DEBUG   consensus/consensus.go:177      received message        {"from": 1}
2020-01-30T15:46:42.711Z        DEBUG   dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:203     received message        {"type": "RecoveryMessage", "from": 1, "height": 6011, "view": 0, "my_height": 6011, "my_view": 0}
2020-01-30T15:46:42.711Z        DEBUG   dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:489     recovery message received       {"dump": {"Type":65,"ViewNumber":0,"Witness":{"invocation":"40ac2ade580abf272a4a0170842fb3d9e8a9914c0ee9bf0380b15d6287e144e6cea0d5ce47762a8c221b79397979351b97e404dcced895f6fe453a4dc02d43b111","verification":"2102103a7f7dd016558597f7960d27c516a4394fd968b9e65155eb4b013e4040406eac"}}}
2020-01-30T15:46:42.712Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
2020-01-30T15:46:42.712Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
2020-01-30T15:46:42.713Z        DEBUG   dbft@v0.0.0-20200130105505-02c208d154bf/check.go:57     not enough to commit    {"count": 1}
2020-01-30T15:46:42.713Z        WARN    dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).onCommit
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:456
github.com/nspcc-dev/dbft.(*DBFT).onRecoveryMessage
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:543
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200130105505-02c208d154bf/dbft.go:242
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:178
```
[three.log](https://github.com/nspcc-dev/dbft/files/4135606/three.log)

**Comment by roman-khimov** [comment]
```
2020-01-31T12:07:17.587Z        WARN    dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:370     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).updateExistingPayloads
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:370
github.com/nspcc-dev/dbft.(*DBFT).onPrepareRequest
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:310
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:234
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
        github.com/CityOfZion/neo-go@/pkg/consensus/consensus.go:201
2020-01-31T12:07:17.587Z        WARN    dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:370     can't validate commit signature
github.com/nspcc-dev/dbft.(*DBFT).updateExistingPayloads
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:370
github.com/nspcc-dev/dbft.(*DBFT).onPrepareRequest
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:310
github.com/nspcc-dev/dbft.(*DBFT).OnReceive
        github.com/nspcc-dev/dbft@v0.0.0-20200131093116-2117a9426d87/dbft.go:234
github.com/CityOfZion/neo-go/pkg/consensus.(*service).eventLoop
```

**Comment by fyrchik** [comment]
Fixed in #22 .
